pkg/cmd/server/api/v1/types.go

```diff
@@ -932,3 +935,19 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string `json:"pluginOrderOverride,omitempty"`
 }
+
+// PodLimitRequestConfig is the configuration for the PodLimitRequest
+// admission controller which overrides user-provided container request/limit values.
+type PodLimitRequestConfig struct {
```

Comment:
Should this be PodContainerLimitRequestConfig instead?  I still have more to read here, but it won't be long until you can set requests/limits on the pod and what is in the container will become largely irrelevant...